### ISSUE TITLE
Snapshots not updating

### ISSUE THREAD

**Issue by GreenBaii** [tsightler/ring-mqtt#598]
### Describe the Bug

My snapshots are not updating. The ring app shows the correct image. However, ring MQTT reports one from three days ago. I can view my cameras live tweak settings etc but don't have an up-to-date snapshot.

### Steps to Reproduce

I have no idea. It started happening after a few days. I tried to unistall and reinstall but same issue,

### Expected Behavior

I expect the snapshot to update every 600 seconds and on motion. The motion bit I understand if it doesn't because its a battery stick up cam and doorbell 2.

### Log Output

```shell
Can you please help me out with this I've followed the instructions and installed the addon run the command I've checked the /config/ directory and sub-directorys and theres no file there. I am willing to provide logs 100% I just need a hand please :)
```

### Screenshots

I would provide an image but as it shows my street I'd rather not but it is the same image everytime for both the back garden stickup camera and the front garden doorbell.

### Config File

```shell
mqtt_url: mqtt://auto_username:auto_password@auto_hostname
mqtt_options: ""
livestream_user: ""
livestream_pass: ""
disarm_code: ""
enable_cameras: true
enable_modes: false
enable_panic: false
branch: addon
debug: ring-*
location_ids: []
```

### Install Type

Home Assistant Addon

### Version

5.2

### Operating System

Home Assistant OS

### Architecture

arm64

### Machine Details

Raspberry Pi

**Comment by GreenBaii** [comment]
Update: Just tried rebooting the cameras and its still showing the same image, I've also tried re authing my ring account. It's probably useful to say that after I uninstalled and reinstalled it started showing the same image before I uninstalled. Ring api seems to be working as I can pull images using the Ring integration.(I only use ring intervention to test and then removed it again to verify everything was still broken)

**Comment by reubznz** [comment]
I've found that restarting HA will potentially prompt for a new snapshot, but it only happens once. 

My floodlight snapshots are working fine, it's just the battery doorbell that appears to be impacted.

**Comment by GreenBaii** [comment]
Yeah, restarting my home assistant doesn't work or prompt for a new snapshot. It seems to be since I updated HA/ring mqtt it's been broken. I have rich notifications on too. Hmmm.

**Comment by GreenBaii** [comment]
![image](https://user-images.githubusercontent.com/17356000/228632800-0030c79f-3ee5-4747-bad4-7afbde83186f.jpeg)![image](https://user-images.githubusercontent.com/17356000/228632854-a804fec0-49e8-43b6-8754-50363997dd1f.png)

It looks like Home assistant updated recently and mosquito broker I roughly remember it starting after mosquito broker updated

**Comment by reubznz** [comment]
The MQTT Broker was the last thing that updated for me, and the timing seems to correlate.

But in saying that, I use an external-to-HA MQTT instance

**Comment by naterator** [comment]
The author [mentioned](https://github.com/dgreif/ring/issues/1187#issuecomment-1485918385) the cause (seemingly an API change at Ring) and a fix over in dgreif/ring#1187. It also appears there's [a commit in the dev branch](https://github.com/tsightler/ring-mqtt/commit/e0446157cd5b34f0e2a64f1be6a8e870c43bba7d) to fix it.

**Comment by tsightler** [comment]
@GreenBaii Without logs, this is an impossible issue to solve.  I know you posted that you just need help to provide logs, but the text already included a link to step-by-step instructions to gather logs and you provided no details on what specifically you needed help with.

As far as motion snapshots, yes, those are likely broken right now, although admittedly, if you don't have a Ring subscription, then batteries cameras won't work for motion snapshots in any case.

Again, posting a bunch of screenshots is mostly useless, only logs are interesting, they provided every detail every time a snapshot is requested and the status, you can see everything that is happening in real-time by monitoring this log.

**Comment by GreenBaii** [comment]
@tsightler I will attempt to get the logs later. Where will the log files be stored once generated? I have an active ring subscription.

**Comment by GreenBaii** [comment]
Just tried to get the logs, when I run the command given it doesn't return anything. I checked every directory and there's no file with the file name specified on the instructions page there is homeassistant.log if that's helpful

**Comment by GreenBaii** [comment]
> The author [mentioned](https://github.com/dgreif/ring/issues/1187#issuecomment-1485918385) the cause (seemingly an API change at Ring) and a fix over in dgreif/ring#1187. It also appears there's [a commit in the dev branch](https://github.com/tsightler/ring-mqtt/commit/e0446157cd5b34f0e2a64f1be6a8e870c43bba7d) to fix it.

Switched to dev branch issue resolved. Must of been the api updates. Thank you all for your time :)

**Comment by tsightler** [comment]
The fix in the dev branch only applies to motion snapshots, you said above that it was happening for all snapshots, both scheduled and motion, so it's somewhat unclear how dev branch could fix scheduled snapshots, but I will close this issue for now since you indicate it is working.  Hopefully I'll be able to push out the dev fixes as a minor update sometime in the next few days.  Thanks!
